src/rules/import-aliases.ts

```diff
@@ -22,9 +37,12 @@ export const rule = createRule({
 		docs: {
 			description: "Avoid using TypeScript's import aliases.",
 		},
+		fixable: "code",
```

Comment:
Oh! Sorry, I should have noticed here - this is not correct. The rule isn't "fixable", it just has suggestions. I'll send a followup PR and file an issue to add linting for this case.

Reply:
Oh thanks!, now I can better understand what that field actually defines.